api-metastore/src/main/java/org/zalando/nakadi/service/SchemaEvolutionService.java

```diff
@@ -95,7 +96,7 @@ private void collectErrorMessages(final ValidationException e,
         }
     }
 
-    public EventType evolve(final EventType original, final EventTypeBase eventType) throws InvalidEventTypeException {
+    public EventType evolve(final EventType original, final EventTypeBase eventType) {
```

Comment:
Even though the exception is a runtime exception we try to "document" that such methods can throw some exception. So it would be nice to see `throws SchemaEvolutionException` (if that's the case).

Reply:
Done